store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
```

Comment:
This validation logic should also be updated in the HBase store. You could add the logic to the Store class instead with a method similar to:
protected ValidationResult  validateSerialiserClasses(final Class<? extends Serialiser> requiredParentSerialiserClass) {
}

Then both the HBaseStore and AccumuloStore classes could call this method.